packages/react-native/Libraries/Components/Keyboard/Keyboard.js

```diff
@@ -114,6 +114,13 @@ class Keyboard {
     );
 
   constructor() {
+    if (Platform.isVisionOS) {
```

Comment:
Can we not create Keyboard.visionOS.js and have it return a noop or throw upon require? It seems like these `if` should be pulled into the Tester app itself, which is what I could expect in end-user app logic as well. If a visionOS app requires this file, and tries to use any methods, the developer probably isn't understanding the platform correctly.

Reply:
It could be a possible solution but I think it's a nice DX to have instructions that this module is not supported on this platform. 

We can wait with merging of this PR till I will implement the Platform.OS (for visionOS) and make it a noop file.

Reply:
wouldn't the warning be annoying when writing cross platform apps?

Reply:
Agree, if we have the `.visionOS` platform file, would also vote for removing warnings

Reply:
There is also way to `warnOnce` as @satya164 said constant warning might be annoying. Looks like react-native-tvos is following this pattern: 

![image](https://github.com/callstack/react-native-visionos/assets/52801365/0e5ae7cc-066d-4df4-8a3e-6a0059eac1f7)